File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {

	int num[1000], n, max = 0;

	float sum = 0;

	cin >> n;

	for (int i = 0;i < n;i++) {
		cin >> num[i];
		sum += num[i];

		if (max < num[i])
			max = num[i];
	}

	sum = sum / max * 100 / n;

	cout << sum;

	return 0;

}

File: 백준/Bronze/1546. 평균/평균.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int N = Integer.parseInt(br.readLine());
        double[] arr = new double[N];
        
        StringTokenizer st = new StringTokenizer(br.readLine());
        double max = Double.parseDouble(st.nextToken());
        arr[0] = max;
        double sum = 0;
        
        
        
        for (int i = 1; i < N; i++) { 
            arr[i] = Double.parseDouble(st.nextToken());
            max = Math.max(max, arr[i]);
        }
        
        for (int i = 0; i < N; i++) {
            arr[i] = arr[i]/max*100;
            sum += arr[i];
        }
        
        System.out.println(sum / N);
        
        br.close();
    }
}